db/migrate/20230503133754_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
+      t.text :text
+
```

Comment:
- Great work on your migration files, however, I would recommend you use `author_id` when referencing the `user` and `post_id` when referencing `post` in your comments table as it is in the ERD diagram 😉